sources/packages/web/src/components/generic/HeaderNavigator.vue

```diff
@@ -1,25 +1,30 @@
 <template>
-  <slot name="title">
-    <div v-if="routeLocation" class="header-title">
-      <a @click="goBack()">
-        <font-awesome-icon :icon="['fas', 'arrow-left']" class="mr-2" />
-        {{ title }}</a
-      >
-    </div>
-    <div v-else class="header-title">
-      {{ title }}
-    </div>
-  </slot>
-  <v-row>
+  <v-row align="end" class="mb-8">
```

Comment:
does `align="end"` matters here? I see both `align-end` and `mb-8` is contradicting. Am i missing anything

Reply:
`align="end"` if forcing the button to be aligned to the bottom and better aligned with the text, as shown below.

**With `align="end"`**
![image](https://user-images.githubusercontent.com/61259237/169164958-68cb4d95-fdcb-4658-ae2b-0045f4a496bb.png)

**Without `align="end"`**
![image](https://user-images.githubusercontent.com/61259237/169165030-bb6b39c9-0b74-47b0-ab6d-87d4a71a74cc.png)

`mb-8` is adding space between the header and the below component.